x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_requirements_page/fleet_server_requirement_page.tsx

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import {
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiForm,
+  EuiLoadingSpinner,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+  EuiLink,
+} from '@elastic/eui';
+import styled from 'styled-components';
+import { FormattedMessage } from 'react-intl';
+
+import { useStartServices } from '../../../hooks';
+
+export const ContentWrapper = styled(EuiFlexGroup)`
+  height: 100%;
+`;
+
+function renderOnPremInstructions() {
+  return (
+    <EuiPanel grow={false} className="eui-textCenter" paddingSize="l">
+      <EuiTitle size="m">
+        <h2>
+          <FormattedMessage
+            id="xpack.fleet.fleetServerSetup.setupTitle"
+            defaultMessage="Add a Fleet Server"
+          />
+        </h2>
+      </EuiTitle>
+      <EuiSpacer size="m" />
+      <EuiText color="subdued" style={{ maxWidth: '420px' }}>
+        <FormattedMessage
+          id="xpack.fleet.fleetServerSetup.setupText"
+          defaultMessage="A Fleet Server is required before you can enroll agents with Fleet. See the Fleet User Guide for instructions on how to add a Fleet Server."
+        />
+      </EuiText>
+      <EuiSpacer size="l" />
+      <EuiForm>
+        <EuiButton
+          iconSide="right"
+          iconType="popout"
+          fill
+          isLoading={false}
+          type="submit"
+          href="https://www.elastic.co/guide/en/fleet/current/index.html"
+          target="_blank"
+        >
+          <FormattedMessage
+            id="xpack.fleet.fleetServerSetup.setupGuideLink"
+            defaultMessage="Fleet User Guide"
+          />
+        </EuiButton>
+      </EuiForm>
+      <EuiSpacer size="m" />
+    </EuiPanel>
+  );
+}
+
+function renderCloudInstructions(deploymentUrl: string) {
+  return (
+    <EuiPanel grow={false} className="eui-textCenter" paddingSize="l">
+      <EuiTitle size="m">
```

Comment:
For both panels, this whole section of content could be simplified with [EuiEmptyPrompt](https://elastic.github.io/eui/#/display/empty-prompt), which includes a max-width.

Reply:
Great I will make the change